examples/pylab_examples/scatter_demo2.py

```diff
@@ -12,7 +12,7 @@
 datafile = cbook.get_sample_data('goog.npy')
 try:
     # Python3 cannot load python2 .npy files with datetime(object) arrays
-    # unless the encoding is set to bytes. Hovever this option was
+    # unless the encoding is set to bytes. However this option was
     # not added until numpy 1.10 so this example will only work with
     # python 2 or with numpy 1.10 and later
     price_data = np.load(datafile, encoding='bytes').view(np.recarray)
```

Comment:
Not having luck with it:
```python
$ ipython3 
In [1]: import numpy as np
In [2]: np.__version__
Out[2]: '1.11.2'
In [3]: i = np.load('lib/matplotlib/mpl-data/sample_data/goog.npy', encoding='bytes')
In [4]: np.save('goog.npy', i)

$ python2
>>> import numpy as np
>>> np.__version__
'1.11.2'
>>> i = np.load('goog.npy')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib64/python2.7/site-packages/numpy/lib/npyio.py", line 406, in load
    pickle_kwargs=pickle_kwargs)
  File "/usr/lib64/python2.7/site-packages/numpy/lib/format.py", line 637, in read_array
    array = pickle.load(fp, **pickle_kwargs)
ValueError: non-string names in Numpy dtype unpickling
>>> i = np.load('goog.npy', encoding='bytes')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib64/python2.7/site-packages/numpy/lib/npyio.py", line 406, in load
    pickle_kwargs=pickle_kwargs)
  File "/usr/lib64/python2.7/site-packages/numpy/lib/format.py", line 637, in read_array
    array = pickle.load(fp, **pickle_kwargs)
ValueError: non-string names in Numpy dtype unpickling
```

What would probably be the most portable is `np.datetime64`, but unfortunately, it appears Matplotlib doesn't handle it as well as `datetime.datetime`. Probably because we didn't depend on a new enough NumPy to carry the converter in-tree.